File: Bronze/10250/10250.java
// [10250] ACM 호텔
import java.util.Scanner;

public class Main {
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);
        int T = sc.nextInt();
        int H, W, N;

        for (int i = 0; i < T; i++)
        {
            H = sc.nextInt();
            W = sc.nextInt();
            N = sc.nextInt();

            // 층수 = 고객 순서 (꼭대기 층인 경우)
            if (N % H == 0)
                System.out.println(H * 100 + (N / H));
            // 그 외
            else
                System.out.println((N % H) * 100 + (N / H) + 1);
        }
    }
}

File: Bronze/10250/10250.cpp
// [10250] ACM 호텔
#include <iostream>
using namespace std;

int main(void) {
    int T;
    cin >> T;
    
    for (int i = 0; i < T; i++) {
        int H, W, N;
        cin >> H >> W >> N;
        
        // 층수 = 고객 순서 (꼭대기 층인 경우)
        if (N % H == 0)
            cout << H * 100 + (N / H) << "\n";
        // 그 외
        else
            cout << (N % H) * 100 + (N / H) + 1 << "\n";
    }
}
